src/App.tsx

```diff
@@ -43,6 +43,7 @@ const App = ({
   hardware = getHardware(),
   machineConfig = machineConfigProvider,
 }: Props) => {
+  // screenReader.disable()
```

Comment:
@eventualbuddha - Is there a better way to disable `screenReader` than adding uncommented version of this line?

Reply:
We should probably make it remember in local storage, or something.